### DISCUSSION TITLE
HW2-Question about finding start/end tokens

### DISCUSSION THREAD

**Discussion by JinkyungJo** [seominjoon/kaist-ai605#25] [Q&A] [answered]
Hello! I have a simple question for problem 2.1.
If the answer appears multiple times in the context, which point should be used as start point?
(e.g., should it be based on the first time it appears?)

Thanks in advance:)

**Comment by seominjoon** [comment, marked as answer]
In SQuAD, the answer's position is uniquely indicated by `answer_start`, so there can't be ambiguity even if the answer text appears multiple times.
